pkg/landscaper/controllers/installations/actuator.go

```diff
@@ -15,32 +15,26 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
-	"github.com/gardener/component-cli/ociclient/cache"
-
 	"github.com/gardener/landscaper/apis/config"
 	lsv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
 	lsv1alpha1helper "github.com/gardener/landscaper/apis/core/v1alpha1/helper"
 	"github.com/gardener/landscaper/pkg/landscaper/blueprints"
 	"github.com/gardener/landscaper/pkg/landscaper/installations"
 	"github.com/gardener/landscaper/pkg/landscaper/operation"
 	componentsregistry "github.com/gardener/landscaper/pkg/landscaper/registry/components"
-	"github.com/gardener/landscaper/pkg/utils"
 	"github.com/gardener/landscaper/pkg/utils/kubernetes"
 )
 
+const (
+	cacheIdentifier = "landscaper-installation-actuator"
```

Comment:
```suggestion
	cacheIdentifier = "landscaper-installation-controller"
```
I would update all cacheidentifer from `actuator` to `controller`. I think actuator is more a technical implementation whereas semantically its rather the controller that uses the cache